locales/locales.json

```diff
@@ -851,6 +851,26 @@
       "currentlyNoEvents": "Güncel bir etkinlik yok."
     }
   },
+  "news": {
+    "de": {
+      "local": "LOKALE NEWS",
+      "readMore": "Weiterlesen >",
+      "currentlyNoNews": "Es gibt aktuell keine Veranstaltungen.",
+      "eNewsNo": "E-news No"
+    },
+    "ar": {
+      "local": "الأخبار المحلية",
+      "currentlyNoNews": "ليست هناك أخبار متوفرة حاليا",
+      "readMore": "< المزيد",
+      "eNewsNo": "E-news No"
+    },
+    "en": {
+      "local": "LOCAL NEWS",
+      "readMore": "Read more >",
+      "currentlyNoNews": "No news is currently available.",
+      "eNewsNo": "E-news No"
+    }
```

Comment:
Task for Integreat team: Add translation for the rest of the languages

Reply:
This will be done by a translation agency, so it is fine to omit other languages for now. However, these changes will have to be moved to the `locales` repo, but I'll take care of that.

Reply:
Do you need more locales for the native app? If so, I could do this together.

Reply:
@klinzo The React Native app is using the same keys.

Reply:
@amusameh I've created a PR in our locales repo. After it is merged, please revert the locales file here and pull them from the locales repo using git subtrees. How to do it is specified here: https://github.com/Integreat/integreat-locales

Reply:
Thanks @klinzo. Will revert these changes.

Reply:
Jfyi: On native we don't use subtrees yet, but there is a open PR which will hopefully be merged tomorrow.